ios/RNSScreenStackComponentView.mm

```diff
@@ -101,6 +102,14 @@ - (void)unmountChildComponentView:(UIView<RCTComponentViewProtocol> *)childCompo
   return _reactSubviews;
 }
 
+- (void)didMoveToWindow
+{
+  [super didMoveToWindow];
```

Comment:
Could you add a brief comment with a link to the original PR adding this change with `_invalidated` so it does not get lost why it was needed?

Reply:
Fixed in [f6d0e18](https://github.com/software-mansion/react-native-screens/pull/1343/commits/f6d0e18ce5f23b4acfbd0710b36b7c349e7ea0c9)